bin/headers.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+failure=0
```

Comment:
This is a flag indicating whether there's any error. I'll change it to something like `has_failure` to make it more meaningful.